langchain4j-open-ai-responses/src/test/java/dev/langchain4j/model/openai/responses/OpenAiResponsesChatModelIT.java

```diff
@@ -0,0 +1,184 @@
+package dev.langchain4j.model.openai.responses;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.model.chat.request.ChatRequest;
+import dev.langchain4j.model.chat.response.ChatResponse;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+/**
+ * Integration tests for OpenAI Responses API POC.
+ *
+ * Key POC demonstrations:
+ * 1. Uses official OpenAI SDK (not custom HTTP client)
+ * 2. Stateless mode with store=false and encrypted reasoning
+ * 3. Response chaining via previousResponseId
+ * 4. ChatModel interface compatibility
+ *
+ * To run: export OPENAI_API_KEY="sk-your-key"
+ */
+@EnabledIfEnvironmentVariable(named = "OPENAI_API_KEY", matches = ".+")
+class OpenAiResponsesChatModelIT {
+
+    private static final String OPENAI_API_KEY = System.getenv("OPENAI_API_KEY");
+
+    @Test
+    void should_generate_response_using_official_sdk() {
+        OpenAiResponsesChatModel model = OpenAiResponsesChatModel.builder()
+                .apiKey(OPENAI_API_KEY)
+                .modelName("gpt-4o")
+                .temperature(0.7)
+                .build();
+
+        ChatRequest request = ChatRequest.builder()
+                .messages(List.of(UserMessage.from("Say 'Hello from Responses API!' and nothing else.")))
+                .parameters(model.defaultRequestParameters())
+                .build();
+
+        ChatResponse response = model.doChat(request);
```

Comment:
```suggestion
        ChatResponse response = model.chat(request);
```

`chat` is the proper public API, `doChat` should not be called directly